tsdb/index.go

```diff
@@ -3,83 +3,12 @@ package tsdb
 import (
 	"bytes"
 	"fmt"
-	"sync"
 
-	"github.com/influxdata/influxdb/models"
+	"github.com/influxdata/influxdb/pkg/lifecycle"
 	"github.com/influxdata/influxdb/query"
 	"github.com/influxdata/influxql"
 )
 
-// SeriesElem represents a generic series element.
```

Comment:
I assume this was unused code. Please confirm.

Reply:
Yep, I could find no usages of it in any repo.